--- vsftpd-2.1.1/sysdeputil.c.orig	2009-05-27 07:48:24.000000000 +0200
+++ vsftpd-2.1.1/sysdeputil.c	2009-05-29 03:07:39.000000000 +0200
@@ -186,6 +186,11 @@
 #undef __FDMASK
 #endif /* VSF_SYSDEP_HAVE_CAPABILITIES */
 
+#ifdef VSF_SYSDEP_HAVE_LINUX_CLONE
+#include <linux/unistd.h>
+#include <errno.h>
+#endif
+
 #if defined(VSF_SYSDEP_HAVE_LINUX_SENDFILE) || \
     defined(VSF_SYSDEP_HAVE_SOLARIS_SENDFILE)
 #include <sys/sendfile.h>
