diff -urNp -x '*.orig' vsftpd-3.0.2.org/vsf_findlibs.sh vsftpd-3.0.2/vsf_findlibs.sh
--- vsftpd-3.0.2.org/vsf_findlibs.sh	2012-03-28 04:17:41.000000000 +0200
+++ vsftpd-3.0.2/vsf_findlibs.sh	2021-10-03 20:23:24.589487997 +0200
@@ -13,7 +13,6 @@ fi
 # Look for PAM (done weirdly due to distribution bugs (e.g. Debian) or the
 # crypt library.
 if find_func pam_start sysdeputil.o; then
-  locate_library /lib/libpam.so.0 && echo "/lib/libpam.so.0";
   locate_library /usr/lib/libpam.so && echo "-lpam";
   locate_library /usr/lib64/libpam.so && echo "-lpam";
   locate_library /lib/x86_64-linux-gnu/libpam.so.0 && echo "-lpam";
