--- vsftpd-2.1.0/ssl.c~	2009-01-09 21:47:05.000000000 +0100
+++ vsftpd-2.1.0/ssl.c	2010-06-03 11:28:01.145540532 +0200
@@ -593,7 +593,7 @@
   str_reserve(p_str, EVP_MAX_MD_SIZE);
   str_empty(p_str);
   str_rpad(p_str, EVP_MAX_MD_SIZE);
-  if (!X509_digest(p_cert, EVP_sha256(), (unsigned char*) str_getbuf(p_str),
+  if (!X509_digest(p_cert, EVP_sha1(), (unsigned char*) str_getbuf(p_str),
                    &num_bytes))
   {
     die("X509_digest failed");
